flw.go

```diff
@@ -61,7 +61,7 @@ func FLWSrvr(servers []string, timeout time.Duration) ([]*ServerStats, bool) {
 
 		// determine current server
 		var srvrMode Mode
-		switch match[10] {
+		switch match[11] {
```

Comment:
Can we have a `zrVer` regexp that does not shift the groups?

Reply:
Hello, I think we can't avoid shift if we are still using regex compile. Maybe we can swicth to parse by line ?